src/main/java/Ui/Superhero.java

```diff
@@ -0,0 +1,130 @@
+package Ui;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import WordList.ToLearn;
+import WordList.VocabList;
+import WordList.Vocabulary;
+import WordList.Deadline;
+import WordList.Event;
+
+public class Superhero {
+
+    VocabList inputList = new VocabList();
+    Scanner choice = new Scanner(System.in);
+    String input;
+    ArrayList<String> inputArray;
+
+    public void startVocabCheck() {
+        this.printWelcomeMessage();
+        do {
+            input = choice.nextLine();
+            inputArray = Parser.readTask(input);
+            switch (inputArray.get(0)) {
+            case "bye":
+                this.printByeMessage();
+                break;
+            case "list":
+                inputList.printList();
+                break;
+            case "todo":
+                Vocabulary newTodoWord = new ToLearn(inputArray.get(1));
+                printInput(newTodoWord.getWord());
+                inputList.appendList(newTodoWord);
+                break;
+            case "deadline":
+                ArrayList<String> deadlineArray = Parser.readTaskDate(inputArray);
+                Vocabulary newDeadlineWord = new Deadline(deadlineArray.get(0), deadlineArray.get(1));
+                printInput(newDeadlineWord.getWord());
+                inputList.appendList(newDeadlineWord);
+                break;
+            case "event":
+                ArrayList<String> eventArray = Parser.readTaskDate(inputArray);
+                Vocabulary newEventWord = new Event(eventArray.get(0), eventArray.get(1));
+                printInput(newEventWord.getWord());
+                inputList.appendList(newEventWord);
+                break;
+            case "mark":
+                try {
+                    int index = Integer.parseInt(inputArray.get(1));
+                    inputList.getList()[index - 1].setDone(true);
+                    this.printMarkMessage(index - 1);
+                } catch (NumberFormatException e){
```

Comment:
Maybe you can add a space between ")" and "{" to make the formatting more consistent (and to follow the good coding format as shown in the picture)
<img width="551" alt="Screen Shot 2022-02-03 at 12 05 19 PM" src="https://user-images.githubusercontent.com/69306031/152279461-01eff779-e85b-4c15-8673-f5d28fb53131.png">